rs/ethereum/cketh/minter/src/eth_rpc_client/mod.rs

```diff
@@ -56,6 +61,13 @@ impl EthRpcClient {
                 EvmRpcClient::builder_for_ic(TRACE_HTTP)
                     .with_providers(providers)
                     .with_evm_canister_id(evm_rpc_id)
+                    .with_min_attached_cycles(300_000_000_000)
+                    .with_override_rpc_config(OverrideRpcConfig {
+                        eth_get_logs: Some(RpcConfig {
+                            response_size_estimate: Some(100 + HEADER_SIZE_LIMIT),
```

Comment:
What's `100`?

Reply:
It's to use the same initial value as when the EVM-RPC canister is not used. See L.183 
```rust
        // We expect most of the calls to contain zero events.
        let results: MultiCallResults<Vec<LogEntry>> = self
            .parallel_call("eth_getLogs", vec![params], ResponseSizeEstimate::new(100))
            .await;
```
The EVM-RPC canister in contrast will start at `1024 + HEADER_SIZE_LIMIT`

Reply:
Thanks for the info! How about introducing constants for all of these magic values in the code?

Reply:
Makes sense, I introduced `ETH_GET_LOGS_INITIAL_RESPONSE_SIZE_ESTIMATE`